service/src/main/java/bio/terra/workspace/service/resource/ValidationUtils.java

```diff
@@ -116,6 +131,48 @@ public static void validateBucketName(String name) {
     }
   }
 
+  /** Validate whether the input URI is a valid GitHub Repo https uri. */
+  public void validateGitRepoCloneUri(String gitUri) {
```

Comment:
The term "clone URI" is confusing in our code. We use the term clone to refer to our own workspace and resource cloning operations. (ref later comment)

Reply:
renamed.